bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/GemsGuideStarWorker.java

```diff
@@ -276,6 +330,18 @@ private GemsGuideStars findGuideStars(ObsContext obsContext, Set<edu.gemini.spMo
         }
     }
 
+    public static Option<GemsGuideStars> findGuideStars(final ObsContext obsContext) throws Exception {
+        try {
+            final Set<edu.gemini.spModel.core.Angle> posAngles = getPosAngles(obsContext);
+            final List<GemsCatalogSearchResults> results = unloggedSearch(GemsGuideStarSearchOptions.DEFAULT, GemsTipTiltMode.canopus, obsContext, posAngles,
+                    scala.Option.empty());
+            final List<GemsGuideStars> gemsResults = GemsResultsAnalyzer.instance().analyze(obsContext, posAngles, results, scala.Option.empty());
```

Comment:
This seems fine, I'm still at loss why this is on the demo so quick, it normally takes a while being quadratic (I think) on the amount of results. Could we test on a more populated field?